examples/client.go

```diff
@@ -102,6 +107,10 @@ func main() {
 		clientOptions = append(clientOptions, realis.SchedulerUrl(url))
 	}
 
+	if clientKey != "" || clientCert != "" || caCertsPath != "" {
```

Comment:
Should the condition be to have both the private key and certificate? What good is one without the other?

```
(clientKey != "" && clientCert != "")
```

Even with such correction it still does leave opportunity for incorrect settings to take effect. It is better to separate the tests for "caCertsPath" and client.

Reply:
Good catch, will fix in follow up.